lerna.json

```diff
@@ -16,7 +16,6 @@
     "packages/worker",
     "packages/pro/packages/pro"
   ],
-  "useWorkspaces": true,
```

Comment:
This has been removed, it uses workspaces by default: https://github.com/lerna/lerna/blob/main/CHANGELOG.md#700-2023-06-08